client/blocks/importer/wordpress/upgrade-plan/constants.tsx

```diff
@@ -40,24 +45,56 @@ export const UpgradePlanHostingTestimonials = [
 	},
 ];
 
+export const defaultHostingDetails = [
+	{
+		title: translate( 'Reduced error rate' ),
+		description: translate( '16% fewer errors' ),
+		icon: shield,
+	},
+	{
+		title: translate( 'Increased speed' ),
+		description: translate( '30% faster' ),
+		icon: trendingUp,
+	},
+	{
+		title: translate( 'Higher availability' ),
+		description: translate( '3% better uptime' ),
+		icon: chartBar,
+	},
+] as Array< { title: string; description: string | ReactNode; icon: JSX.Element } >;
+
 export function useUpgradePlanHostingDetailsList() {
```

Comment:
Feels strange to have a hook inside `constants.tsx`. Let's move this to the hook folder as well!

Reply:
I moved this into its file and refactored the usage of the hook.